### PR TITLE
Update update_dep to allow passing in the dep to update

### PR DESCRIPTION
**Pull request by tas50**
We need to know the dep we're updating throughout this script and if
it's not a gem we don't know the name. Allow passing in the name via arg

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -211,8 +211,8 @@ subscriptions:
   # NOTE: The branch of Ohai here needs to be updated when setting up a stable branch of chef/chef
   - workload: chef/ohai:master_completed:pull_request_merged:chef/ohai:master:*
     actions:
-      - bash:.expeditor/update_dep.sh
+      - bash:.expeditor/update_dep.sh ohai
```
Comment (nkierpiec) [inline .expeditor/config.yml:214]: Need to remove the `ohai` here

File: .expeditor/config.yml [hunk 2]
```diff
@@ -211,8 +211,8 @@ subscriptions:
...
   # NOTE: When the stable branch of chef/chef is being cut you probably want to remove this subscription
   - workload: chef/chefstyle:master_completed:pull_request_merged:chef/chefstyle:master:*
     actions:
-      - bash:.expeditor/update_dep.sh
+      - bash:.expeditor/update_dep.sh chefstyle
```
Comment (nkierpiec) [inline .expeditor/config.yml:218]: Need to remove the `chefstyle` here

**Review by nkierpiec — CHANGES_REQUESTED** [review]
Small fixes to the config then you should be golden

**Review by nkierpiec — APPROVED** [review]

**Comment by tduffield** [comment]
So the only issue here is that `pull_request_merged` does not have an `EXPEDITOR_VERSION` variable. You'll likely need to fetch that from the VERSION file via a curl command or have a mode of operating where it does not exist.

**Pull request merged**